packages/blade/src/components/Form/Selector/SelectorInput.web.tsx

```diff
@@ -30,7 +30,8 @@ const getHoverStyles = ({
 
   return {
     borderColor: borderColor ? getIn(theme, borderColor) : undefined,
-    backgroundColor: getIn(theme, backgroundColor),
+    backgroundColor:
+      backgroundColor === 'transparent' ? 'transparent' : getIn(theme, backgroundColor),
```

Comment:
We don't need this. 

We have `color.transparent` token in our theme file now.

Reply:
fixed